pkg/aws/client.go

```diff
@@ -98,17 +106,39 @@ func (c *r53) resourceRecordSet(record *phonebook.DNSRecord) *types.ResourceReco
 		// information about this.
 		//
 		// 1. https://docs.aws.amazon.com/Route53/latest/APIReference/API_AliasTarget.html
+
 		set.AliasTarget = &types.AliasTarget{
 			DNSName:      &record.Spec.Targets[0],
 			HostedZoneId: &hostedZoneID,
 		}
-
-		return &set
+		// Note: For Alias records, TTL is not used and should be omitted
+		set.TTL = nil
+	} else {
+		// Handle different record types
+		switch types.RRType(record.Spec.RecordType) {
+		case types.RRTypeA, types.RRTypeAaaa, types.RRTypeCname, types.RRTypeTxt:
```

Comment:
`RRTypeTxt` behave differently than the others here. But it's fine to keep it here as I'm working on a PR to implement some changes for DNS-01 solver.

Reply:
No problem, I'll leave it as it is for now